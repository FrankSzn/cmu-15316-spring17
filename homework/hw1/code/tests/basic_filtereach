as principal admin password "admin" do
   set records = []
   append to records with { name = "Alice", date = "5-14-13" }
   append to records with { name = "Bob", date = "3-1-14" }
   append to records with { name = "Mallory", date = "11-8-16" }
   set names = records
   filtereach rec in names with equal(rec.name, "Tamale")
   return names
***