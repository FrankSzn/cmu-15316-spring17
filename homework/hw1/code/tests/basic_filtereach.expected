{"status":"SET"}
{"status":"APPEND"}
{"status":"APPEND"}
{"status":"APPEND"}
{"status":"SET"}
{"status":"FILTEREACH"}
{"output":[],"status":"RETURNING"}
